### PR TITLE
Add MacType 2021.1-rc1 version

### PR DESCRIPTION
**Pull request by elricli**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17558)

### CODE REVIEW

File: manifests/m/MacType/MacType/2021.1-rc1/MacType.MacType.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+PackageIdentifier: MacType.MacType
+Publisher: MacType
```
Comment (ItzLevvie) [inline manifests/m/MacType/MacType/2021.1-rc1/MacType.MacType.yaml:2]:
```suggestion
Publisher: FlyingSnow, Samantha Glocker
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/m/MacType/MacType/2021.1-rc1/MacType.MacType.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+PackageName: MacType
+PackageVersion: 2021.1-rc1
```
Comment (ItzLevvie) [inline manifests/m/MacType/MacType/2021.1-rc1/MacType.MacType.yaml:4]:
```suggestion
PackageVersion: 2021.1-RC1
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
You'll have to update the version folder to fix the casing.

**Comment by drrrMikado** [comment]
> You'll have to update the version folder to fix the casing.

Thanks

File: manifests/m/MacType/MacType/2021.1-rc1/MacType.MacType.yaml (hunk 1 above)
Comment (drrrMikado) [reply manifests/m/MacType/MacType/2021.1-rc1/MacType.MacType.yaml]: I think it is not necessary to modify this, just like `PicGo/PicGo`.

Comment (ItzLevvie) [reply manifests/m/MacType/MacType/2021.1-rc1/MacType.MacType.yaml]: Windows Package Manager relies on Control Panel's data and it has to match so others can use the `winget list` and `winget upgrade` commands.

File: manifests/m/MacType/MacType/2021.1-RC1/MacType.MacType.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+PackageIdentifier: MacType.MacType
+Publisher: MacType
```
Comment (ItzLevvie) [inline manifests/m/MacType/MacType/2021.1-RC1/MacType.MacType.yaml:2]:
```suggestion
Publisher: FlyingSnow, Samantha Glocker
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/m/MacType/MacType/2021.1-RC1/MacType.MacType.yaml (hunk 3 above)
Comment (drrrMikado) [reply manifests/m/MacType/MacType/2021.1-RC1/MacType.MacType.yaml]: Thanks, It's fixed.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**